src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Lottery {

```

Comment:
The main method and getRandomBall must be in different classes. Also, the logic for getting a Ball object with random parameters should be in the getRandomBall method and not in the constructor